src/routes/safe/store/actions/fetchSafe.ts

```diff
@@ -39,8 +39,8 @@ const buildOwnersFrom = (
     })
   })
 
-const buildModulesLinkedList = (modules: string[], nextModule: string): Array<ModulePair> | null => {
-  if (modules.length) {
+const buildModulesLinkedList = (modules: string[] | null, nextModule: string): Array<ModulePair> | null => {
```

Comment:
I'll use `undefined` as chaining operator returns `undefined` and you're using it in the line 114

> The ?. operator functions similarly to the . chaining operator, except that instead of causing an error if a reference is nullish (null or undefined), the expression short-circuits with a return value of undefined.